operations/orderitems.md

```diff
@@ -314,22 +314,24 @@ Returns all order items. At least one of the `OrderItemIds`, `ServiceOrderIds`,
 | `ClosedUtc` | string | optional | Date and time of the item bill closure in UTC timezone in ISO 8601 format. |
 | `StartUtc` | string | optional | Start of the order item in UTC timezone in ISO 8601 format. |
 | `ClaimedUtc` | string | optional | Date and time when the order item was claimed in UTC timezone in ISO 8601 format. |
-| `AccountingState` | [Accounting State](accountingitems#accounting-item-state) | required | Accounting state of the order item. |
-| `Type` | [Order item type](#order-item-type) | required | Order item type, e.g. whether product order or space order. |
-| `Options` | [Order item options](#order-item-options) | required | Options of the order item. |
-| `Data` | [Order item data](#order-item-data) | optional | Additional order item data. |
-
-#### Account type
-
-* `Company`
-* `Customer`
+| `AccountingState` | [Order item accounting state](orderitems.md#order-item-accounting-state) | required | Accounting state of the order item. |
+| `Type` | [Order item type](orderitems.md#order-item-type) | required | Order item type, e.g. whether product order or space order. |
+| `Options` | [Order item options](orderitems.md#order-item-options) | required | Options of the order item. |
+| `Data` | [Order item data](orderitems.md#order-item-data) | optional | Additional order item data. |
 
 #### Revenue type
 
 * `Service`
 * `Product`
 * `Additional`
 
+#### Order item accounting state
```

Comment:
This was overridden before to point to accounting items page, do you want to move it here?

Reply:
It's 'Order item accounting state' so I'll roll with it.